src/partials/proposal.html

```diff
@@ -37,7 +37,7 @@ <h2 class="visually-hidden">English Advantages</h2>
                 skills and confidence grow with every session.</p>
             </li>
           </ul>
-          <button type="button" class="button proposal-section-button">Leave an application</button>
+          <a type="button" class="button proposal-section-button" href="#application">Leave an application</a>
```

Comment:
'a' tag does not have `type` attribute, please remove it

Reply:
done